src/styles/reset.css

```diff
@@ -0,0 +1,16 @@
+/* code here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body,
+h2,
+h3,
+p {
```

Comment:
Why normalize h2, h3 and if you didn't use it?

Reply:
for a future :)